src/CustomElementInternals.js

```diff
@@ -42,6 +42,20 @@ export default class CustomElementInternals {
   }
 
   /**
+   * @param {!Node} node
+   * @return {!CustomElementDefinition|null}
+   */
+  nodeToDefinition(node) {
+    const name = typeof node.__CE_is === 'string' ? node.__CE_is : node.localName;
```

Comment:
`__CE_is` is a new variable I have created on nodes that are customized builtin elements.